tests/unit/parsers/test.manifestjson.js

```diff
@@ -874,10 +936,47 @@ describe('ManifestJSONParser', () => {
 
       expect(manifestJSONParser.isValid).toEqual(true);
       const { warnings } = addonLinter.collector;
-      expect(warnings[0].code).toEqual(messages.MANIFEST_CSP_UNSAFE_EVAL.code);
+      expect(warnings[0].code).toEqual(messages.MANIFEST_CSP_UNSAFE_EVAL);
       expect(warnings[0].message).toEqual(
-        "Using 'eval' has strong security and performance implications."
+        messages.manifestCspUnsafeEval('content_security_policy').message
+      );
+
+      // Clear any warnings and errors collected.
+      addonLinter.collector.warnings = [];
+      addonLinter.collector.errors = [];
+
+      // Manifest v3 formats.
+      const contentSecurityPolicy = {
+        extension_pages: invalidValue,
+        content_scripts: invalidValue,
+        // Alias for content_scripts.
+        isolated_world: invalidValue,
+      };
+
+      const jsonV3 = validManifestJSON({
+        content_security_policy: contentSecurityPolicy,
+        applications: {
+          // The new content_security_policy syntax is only supported
+          // on Firefox >= 72.
+          gecko: { strict_min_version: '72.0' },
```

Comment:
Do we actually issue a warning / error if `strict_min_version` is `71.0`? I don't think we do as part of the schema, but maybe as part of mdn-browser-compat-data?

Reply:
it is coming from the rules implemented based on the metadata coming from `mdn-browser-compat-data`.